### DISCUSSION TITLE
Proposal: <podcast:primaryMedium>

### DISCUSSION THREAD

**Discussion by agates** [Podcastindex-org/podcast-namespace#620] [Tag Proposal]
## Primary Medium tag for publisher mediums
`<podcast:primaryMedium>`<br><br>
This elements specifies a primary medium for UX purposes for feeds using the `publisher` medium.

This removes ambiguity for applications that want to display the publisher as a dedicated page.  Without this, applications have to guess which medium to use to interpret the publisher's primary content type based on the referenced feeds.  While guessing might be straightforward for publishers with only one referenced medium, some publishers may use a variety of mediums.  With `<podcast:primaryMedium>`, the publisher tells the app which medium is most relevant for them.

### Parent
&nbsp; `<channel>`

### Count
&nbsp; Single

### Node Value
&nbsp; The node value is a string denoting one of the values from `<podcast:medium>`.

### Example (music publisher with a podcast)
```xml
<rss xmlns:podcast="https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md" version="2.0">
    <channel>
        <title>X</title>
        
        <podcast:guid>05c97365-89a6-41f1-bef4-e1f9e001cbed</podcast:guid>
        
        <podcast:medium>publisher</podcast:medium>

        <podcast:primaryMedium>music</podcast:primaryMedium>
        
        <podcast:remoteItem
            feedGuid="35b7abaf-f93c-480b-a1c1-4d79069a895b"
            medium="music"
        />
        <podcast:remoteItem
            feedGuid="d3e9aff6-c380-4660-bc8c-5a010ea21ad1"
            medium="music"
        />
        
        <podcast:remoteItem
            feedGuid="e67244e7-943b-4345-96c4-8422fb5206b0"
            medium="podcast"
        />
    </channel>
</rss>
```

**Comment by Dwev** [comment]
Is this to help the app render the correct labels, like "album" instead of "podcast" or something?

- Why can't the app just determine the primary medium from the referenced sources?
- In your example, should the podcast feed also be rendered as a music feed because it is under the primary medium of music?

**Comment by theDanielJLewis** [comment]
Why not simply take the first instance as the primary?

That's how most podcast apps already work with categories.

**Comment by MerryOscar** [comment]
@agates personally I don't think this is necessary because:

1. The way most listeners will land on a publisher feed page in an app is via the child feed which will have a `medium` tag so there is no need for the app logic to guess it can just use the `medium` of the child feed:
    - if I discover Ainsley Costello's publisher feed from one of her music tracks - the app knows to render the feed page as an Artist
    - if I discover Adam Curry's publisher feed via the Podcasting 2.0 podcast - the app knows to render the feed page as a Publisher (or something else based on the app's preferred wording)
  

  
2. Publishers may publish both music and a podcast - and in these cases rather than defaulting to a primary medium it would be better to render the publisher feed page based on the above logic where the child medium determines what the page looks like

3. Determining the medium from the referenced child feeds is relatively trivial for an app

**Reply by agates** [reply]
1. This is an invalid assumption.  Your focus is only for your app, and your anti-competitive behavior is irrelevant.  This tag is about removing assumptions.
2. This point is invalid because point one is invalid.
3. This tag is about removing assumptions.

This is such an easy tag compared making people guess.  Why do you like guessing?

**Reply by MerryOscar** [reply]
> _"your anti-competitive behavior is irrelevant."_

What behaviour of mine is anti-competitive? I am trying to have a discussion with you in good faith about the best way to push open podcasting forward - I spent the time coming up with the proposal for publisher feeds with the goal of making podcasting more open and the experience better for listeners on **_all apps_**. We could have easily built artist pages into Fountain but thought it was better to find an open way of doing it with the rest of the community. How is that anti-competitive?

Anyway as I said my comment on this was in good faith and happy to continue the discussion - I don't mind if this becomes part of the spec I can see it's use just sharing my thoughts to be helpful 🤷‍♂️

**Reply by jamescridland** [reply]
@agates We're all in this together, mate. Calm yourself.
